tests/integration-tests/configs/release-process.yaml

```diff
@@ -0,0 +1,14 @@
+{%- import 'common.jinja2' as common with context -%}
+---
+test-suites:
+  basic:
+    test_essential_features.py::test_essential_features:
```

Comment:
Can we add a comment here explaining that 

`we're verifying that aws-parallelcluster package is available in the 3 partitions, that is possible to create a cluster and essential features are working`